testing-common/src/main/java/io/opentelemetry/instrumentation/testing/LibraryTestRunner.java

```diff
@@ -91,10 +97,8 @@ private LibraryTestRunner() {
   @Override
   public void beforeTestClass() {
     // just in case: if there was any test that modified the global instance, reset it
-    if (GlobalOpenTelemetry.get() != openTelemetry) {
```

Comment:
I removed this check because I think it will always be true because `GlobalOpenTelemetry` wraps the instance used in `set`.